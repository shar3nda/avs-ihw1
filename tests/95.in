13
5 1 9 5 4 -1 2 -1 1 -1 4 2 7