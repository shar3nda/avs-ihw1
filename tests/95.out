-2 0 -2 -3 1 5 0 -4 6 10